### DISCUSSION TITLE
USB Voltage solution

### DISCUSSION THREAD

**Discussion by KieranKPRichards** [mriscoc/Ender3V2S1#1165] [Ideas]
Good day! 

I have a suggestion with regards to the USB Voltage issue. Instead of manually taping the pin, why not control the USB power delivery from the Pi itself? You can make use of the package UHUBCTL to control power delivery to all ports on the Pi. 

Hope this is helpful.

**Comment by TheWebMachine** [comment]
It was my understanding - based on UHUBCTL compat list and my own limited excursion down this particular rabbit hole - that Pi's have the ports ganged, requiring powering down multiple ports at a time, instead of individually. This makes use of cams, touchscreens, and other peripherals difficult (and finicky in my experience). I mean, if the printer is the only thing plugged in, then I'm sure it is fine, but it kinda criples the Pi from being a more robust OctoPrint/Pi setup on its own. 

A simple power decoupler is cheap to buy and even cheaper to make yourself if you're crafty with a knife and have an extra cable laying around. 🤷🏻‍♂️
